.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+- '2.7'
+install:
+- pip install -r requirements_dev.txt
+script:
+- flake8
+- nosetests
+after_script:
+- codecov -u https://codecov.io --no-fail
+deploy:
+  provider: pypi
+  user: wesleybalvanz-wf
+  password:
+    secure: sFB1+BIdh4KDs+PuR+wAxOI0abVse1YEzSg04UZ/jGkuJeAxo2pCLFFzo5cwWEv6T9mVN1lVoC4ZnLfD0mkasXoq3mdATHX9uik8F4B7dQR+Azyl/uS1y/3eAxCOUkC6dSQj3USoWXJWf6dDbRYWcMgMhSUSylx3Dg2GH1xYlrEpdzydNXWIICTNTFURtO7QmYXlVSncjJhSWaKmkTikulgtcklwIp1GrLd7NkQeLMHpeksOLRgJLDA5PV/FopYNV4aeREkyDaGwWul73zwTWp6uriX6G2+Kk+JLco5IP5Jm60t5Xi/c7dBiVBb2JgEr5lXPyZWu4NBdNsJNXcOTc6mabjzBGr4O2u0IvTN21VpEfaW2TVQqY6qpDaCmqVjsWewdZAHuNzfddeNcxRy/JWOaD+2PfVXnCnfwt7QvM+LAkPE4rmSF/j8yA9Meyfyg9PWiFZVzhKIe/gGd6Pio8/EkLJLL5GuQDJcGtE+nuekNpGyKoK15LcVqhQBU5WLivPPLJ3U5e4RE7A/k23xhjF4V0r8Il6zWDdRazvYvsJL0KnvcjQY6pFW0bTi7H3KSzcTCIET9+d3RhqVKVb8S/6UEiUegC6fLowtL667PO6Dq2vjGFVMn4jcM9PfVQoT6HDn3aF0+ge7rG4gFKbyoWTGA/pma1WqkD3US2rRVNIc=
```

Comment:
"I know - lets have them copy-paste their password and make it publicly available." - TravisCI
...did these guys happen to previously work at Sony?